aries_vcx/src/messages/a2a/mod.rs

```diff
@@ -319,7 +335,9 @@ impl A2AMessage {
     const QUERY: &'static str = "query";
     const DISCLOSE: &'static str = "disclose";
     const BASIC_MESSAGE: &'static str = "message";
-    const OUT_OF_BAND: &'static str = "out-of-band";
+    const OUT_OF_BAND_INVITATION: &'static str = "invitation";
```

Comment:
dtto about naming, I'd go with the original value

Reply:
This is constrained by the RFC.